docs/reference/configuration.md

```diff
@@ -65,6 +65,8 @@ This reference uses `.` to denote the nesting of values.
     + `database.cosmosdb.endpoint` `(string : "")` - CosmosDB account endpoint, e.g. `https://<account>.documents.azure.com/`.
     + `database.cosmosdb.database` `(string : "")` - CosmosDB database name.
     + `database.cosmosdb.container` `(string : "")` - CosmosDB container name.
+    + `database.cosmosdb.throughput` `(uint : 400)` - The container's RU/s.
+    + `database.cosmosdb.autoscale` `(bool : false)` - If set, container throughput is autoscaled (requires a minimum of 1000 RU/s). Otherwise, uses "Manual" mode ([Docs](https://learn.microsoft.com/en-us/azure/cosmos-db/provision-throughput-autoscale)).
```

Comment:
1. Suggest to remove the "requires..." and keep the link as the azure docs may change 
2. include "CosmosDB..." as part of the parameter description

Reply:
Done